doc/tutorial/single_cell_allen.rst

```diff
@@ -0,0 +1,170 @@
+.. _tutorialsinglecellallen:
+
+A single cell model from the Allen Brain Atlas
+==============================================
+
+In this tutorial we'll see how we can take a model from the `Allen Brain Atlas <https://brain-map.org/>`_
+and run it with Arbor.
+
+.. Note::
+
+   **Concepts covered in this example:**
+
+   1. Take a model from the Allen Brain Atlas.
+   2. Load a morphology from an ``swc`` file.
+   3. Load a parameter fit file and apply it to a :class:`arbor.decor`.
+   4. Building a :class:`arbor.cable_cell` representative of the cell in the model.
+   5. Building a :class:`arbor.recipe` reflective of the cell stimulation in the model.
+   6. Running a simulation and visualising the results.
+
+Obtaining the model
+-------------------
+
+We need a model for which morphological data is available. We'll take
+`a cell from the mouse's visual cortex <http://celltypes.brain-map.org/experiment/electrophysiology/488683425>`_.
+As the ``README`` in the "Biophysical - all active" model (zip) file explains, the data set is created for use with the Allen SDK and the
+Neuron simulator. Instructions about running the model with the
+Allen SDK and Neuron `can be found here <https://allensdk.readthedocs.io/en/latest/biophysical_models.html>`_.
+For your convenience, this tutorial comes with a reference trace pre-generated.
+
+In the "Biophysical - all active" model (zip) file you'll find:
+
+* a directory with mechanisms (modfiles). For your convenience, Arbor already comes with the
+  :func:`arbor.allen_catalogue`, containing mechanisms used by the Allen Brain Atlas.
+* an ``swc`` file describing the morphology of the cell.
+* ``fit_parameters.json`` describing a set of optimized mechanism parameters. This information will need to be
+  loaded into the Arbor simulation.
+
+We will replicate the "Sweep 35" experiment, which applies a current of 150 nA for a duration of 1 s.
```

Comment:
Style: put quantities in some markup.

Reply:
I prefer to reserve code-like markup for code, filenames, variable names, and use good old quotes as they were intended. Or do you have some other markup in mind?

Reply:
Isn't there a `math` role we have? At least in Sphinx? Would tie in nicely with usage in LaTeX

Reply:
There is [a math role](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#math)! But are names math? "Sweep 35" is the name of an experiment, which I'd write like that in a paper too. I guess we could think of it as a variable name (it is also a row in a table) if you insist on changing it from quotes.

Reply:
I was thinking about $150 nA$

Reply:
Ach so. Nah, that's a TeXism. I think there's nothing unclear about how it is now, numeric characters may appear in text, nothing wrong with that ;)

Reply:
It was a comment on style and it's not tied to TeX, but to proper typesetting.

Reply:
Inline number should conform to the typesetting of the text they appear in, I would say.